tests/epyccel/recognised_functions/test_numpy_types.py

```diff
@@ -250,7 +250,7 @@ def test_numpy_int_scalar(language, function_boundaries):
 
     get_int = function_boundaries[0]
     # Modifying a global variable in a scop will change it to a local variable, so it needs to be initialized.
```

Comment:
Should "scop" read "scope" here?

Reply:
Added to detectable typos : [56c1d6b](https://github.com/pyccel/pyccel/pull/2284/commits/56c1d6bf0fee5dbdfb21879b0b95549694044c38)
Fixed : [b36b641](https://github.com/pyccel/pyccel/pull/2284/commits/b36b641f96d2e58f9977aa318befb05deb7a1155)